src/components/common/BalanceChannelsButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { SwapOutlined } from '@ant-design/icons';
+import styled from '@emotion/styled';
+import { Button } from 'antd';
+import { usePrefixedTranslation } from 'hooks';
+import { useStoreActions } from 'store';
+import { Network } from 'types';
+
+const Styled = {
+  Button: styled(Button)`
+    margin-left: 8px;
+  `,
+};
+
+interface Props {
+  network: Network;
+}
+
+const BalanceChannelsButton: React.FC<Props> = ({ network }) => {
+  const { l } = usePrefixedTranslation('cmps.common.BalanceChannelsButton');
+  const { showBalanceChannels, resetBalanceChannels } = useStoreActions(s => s.modals);
+
+  return (
+    <Styled.Button
+      onClick={() => resetBalanceChannels(network).then(showBalanceChannels)}
+    >
+      <SwapOutlined /> {l('btn')}
```

Comment:
Please remove the inline text and show the message in a tooltip, similar to the `SyncButton`.